### PR TITLE
Improve the WebUI command

### LINKED ISSUE #2676: oq webui start not working as expected
I just installed oq 2.3 via apt on ubuntu 16.04. I wanted to try the api so I ran

    oq webui start

which just gave me

"/usr/bin/python: No module named openquake.server"

### PR DESCRIPTION
**Pull request by daniviga**
This PR improves the behavior of the `oq webui` command.

Now it can be used in both single and multi user environment; in a multi-user environment it does tasks only if the command is run by the same user that is the database owner.

The `oq` entry point has been improved too. Now, when OQ is installed in `/opt` (packages), the `sys.path` is saved inside `os.environ['PYTHONPATH']` to pass the proper configuration to any subproccess call.

Tests (change affects only linux + packages): https://ci.openquake.org/job/zdevel_oq-engine/2450/

Fixes #2676. Thanks to @3rdcycle for the feedback.

### CODE REVIEW

File: openquake/commands/webui.py [hunk 1]
```diff
@@ -34,8 +37,13 @@ def webui(cmd, hostport='127.0.0.1:8800'):
     start the webui server in foreground or perform other operation on the
     django application
     """
-    dbserver.ensure_on()  # start the dbserver in a subproces
+
+    db_owner = os.path.expanduser(config.get('dbserver', 'file'))
+    if not os.access(db_owner, os.W_OK):
```
Comment (micheles) [inline openquake/commands/webui.py]: I would use the name `db_path` not `db_owner`

File: bin/oq [hunk 2]
```diff
@@ -26,6 +26,7 @@ if os.path.realpath(__file__) == '/opt/openquake/bin/oq':
     # https://docs.python.org/2/library/sys.html#sys.path
     sys.path.insert(1, '/opt/openquake/lib/python' + str(sys.version_info[0]) +
                     '.' + str(sys.version_info[1]) + '/site-packages')
+    os.environ['PYTHONPATH'] = ":".join(sys.path)
```
Comment (micheles) [inline bin/oq]: How does this work on Windows?
BTW, I would write

```python
 sys.path.insert(1, '/opt/openquake/lib/python%d.%d/site-packages' % sys.version_info[:2])

**Comment by micheles** [comment]
We could add in the packager a call to `oq webui` to test the message about not being the right user.

File: bin/oq (hunk 2 above)
Comment (daniviga) [inline bin/oq]: On Windows we never enter that if, so this code will never executed (anyway, `os.environ` + `PYTHONPATH` works too, we use it in `.bat` entry points).

**Comment by daniviga** [comment]
New tests: https://ci.openquake.org/job/zdevel_oq-engine/2451/

**Comment by daniviga** [comment]
https://ci.openquake.org/job/zdevel_oq-engine/2453/

**Comment by daniviga** [comment]
Tests are green: https://ci.openquake.org/job/zdevel_oq-engine/2454/

**Comment by micheles** [comment]
Please update the changelog. For the rest, LGTM

**Pull request merged**